.github/workflows/main.yml

```diff
@@ -0,0 +1,30 @@
+name: Ruby
+
+on:
+  push:
+    branches:
+      - master
+
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Ruby ${{ matrix.ruby }}
+    strategy:
+      matrix:
+        ruby:
+          - '3.0.6'
+          - '3.1.4'
+          - '3.2.2'
+          - '3.3.0'
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+        bundler-cache: true
+    - name: Run the default task
+      run: bundle exec rake
```

Comment:
need a bundle install here, no?

Reply:
Nope - lines 24 to 28 handle that

Reply:
You can see in the action that `bundle install` is running:
https://github.com/kickstarter/mail-x_smtpapi/actions/runs/7687544862/job/20947697441?pr=9#step:3:27